src/util/formHelper.ts

```diff
@@ -152,32 +182,31 @@ export const learningResourceRules = {
   },
 };
 
-export const topicArticleRules = {
+export const topicArticleRules: RulesType<TopicArticleFormType> = {
   ...formikCommonArticleRules,
   visualElementAlt: {
     required: false,
-    onlyValidateIf: values => values.visualElement && values.visualElement.resource === 'image',
+    onlyValidateIf: values =>
+      isEmbed(values.visualElement[0]) && values.visualElement[0].data.resource === 'image',
   },
   visualElementCaption: {
     required: false,
     onlyValidateIf: values =>
-      values.visualElement &&
-      (values.visualElement.resource === 'image' || values.visualElement.resource === 'brightcove'),
+      isEmbed(values.visualElement[0]) &&
+      (values.visualElement[0].data.resource === 'image' ||
+        values.visualElement[0].data.resource === 'brightcove'),
   },
 };
 
-export const parseImageUrl = metaImage => {
+export const parseImageUrl = (metaImage?: MetaImage) => {
   if (!metaImage || !metaImage.url || metaImage.url.length === 0) {
-    if (metaImage?.id) {
-      return metaImage.id;
-    }
```

Comment:
Trengs ikkje denne sjekken?

Reply:
`transformArticleFromApiVersion` konverterte tidligere `MetaImage` til `{id: string; alt: string}` der ID ble satt på lignende måte som litt lenger ned i denne funksjonen.

Reply:
Dette burde dog testes, da jeg vet det er forskjell på null og undefined for metaImage.